client/src/components/PublicPages/AuditPlanner/AuditPlanner.test.tsx

```diff
@@ -1,17 +1,23 @@
 import React from 'react'
 import userEvent from '@testing-library/user-event'
-import { screen, waitFor, within } from '@testing-library/react'
+import { QueryClientProvider } from 'react-query'
+import { fireEvent, screen, waitFor, within } from '@testing-library/react'
```

Comment:
Reviewing with [`?w=1`](https://github.com/votingworks/arlo/pull/1597/commits/2c3f94feae9363e7005720bed148813084cafd28?w=1) recommended for this commit!